### PR TITLE
Week 2 assignments

### PR DESCRIPTION
**Pull request by adnan-dedhawala**
### Assignment 1

I have created an HTML page for displaying articles. It has the following features:

-  responsive layout
-  a header section with a title and subtitle
- a responsive container that has 2 column layout. the right side is the articles and the left side is the main article
- the main article has an image, title and text
- side articles have a title, text and a link element
Screenshot 2022-10-09 at 6 54 44 AM
- the two-column layout is responsive and creates a single-column layout when the screen width is less than 768px
Screenshot 2022-10-09 at 7 02 29 AM
- the page has a footer section
Screenshot 2022-10-09 at 7 02 40 AM
- the page has a form at the bottom with client-side validation implemented.
Screenshot 2022-10-09 at 7 38 11 AM

### Assignment 2
- Created a div with the text "Hi" and aligned it to the centre using flexbox.
- Created 4 divs (red, yellow, green and blue) and using position absolute aligned them as follows
Screenshot 2022-10-09 at 9 32 39 AM
- using CSS transform property changed the position and added rotation to the divs
- using transition property added delay of 1.5 seconds to achieve the desired result
Screenshot 2022-10-09 at 9 32 45 AM

### CODE REVIEW

File: Week-2/2.2/style.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+.window {
+  width: 100px;
+  height: 100px;
+  position: absolute;
+  transition: transform 1.5s;
+}
+
+.one {
+  top: 0;
+  left: 0;
```
Comment (arqam-pesto) [inline Week-2/2.2/style.css:54]: these can be concluded in a single css and re-use the same class.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "version": "1.0.0",
+  "description": "Welcome to Pesto assignments.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "npm-run-all -p format:*",
+    "format:code": "npm run prettier:code -- --write",
+    "format:other": "npm run prettier:other -- --write",
+    "prettier:code": "prettier \"**/*.{ts,tsx,js,jsx}\"",
+    "prettier:other": "prettier \"**/*.{css,json,scss,html}\""
```
Comment (arqam-pesto) [inline package.json]: no need of this, also try and add git hooks with husky

File: package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "p6-arqam-adnan-dedhawala",
+  "version": "1.0.0",
+  "description": "Welcome to Pesto assignments.",
```
Comment (arqam-pesto) [inline package.json:4]: add author name

**Review by arqam-pesto — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (adnan-dedhawala) [reply package.json:4]: done

File: package.json (hunk 2 above)
Comment (adnan-dedhawala) [reply package.json]: done

**Pull request merged**